### PR TITLE
5 Real Life Use Cases of 5G Ultra-Reliable Low-Latency Communication (URLLC)

### LINKED ISSUE #1004: 5 Real Life Use Cases of 5G Ultra-Reliable Low-Latency Communication (URLLC)
### Brief Summary: 
Describe the what, why, and how of your content idea in 2-5 sentences.

URLLC is one of the three facets of 5G, alongside MMTC (Massive Machine Type Communication) and eMBB (enhanced Mobile Broadband). In particular, URLLC features two parts; ultra-reliability, which constitutes network reliability of beyond 99.999%, coupled with very low latency for packet transmission. This article discusses real-life use cases of 5G URLLC.
### Key Takeaways: 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?

1. What is 5G URLLC?
2. Application of 5G URLLC in:

 

- Smart Factory/Industrial Automation
- Healthcare industry
- Intelligent transportation
- Entertainment 
- Smart Electricity grid and harbor automation

### References: 
Please list links to any published content/research that you intend to use to support/guide this article. (If none, please indicate 'N/A'.)

### PR DESCRIPTION
**Pull request by EKahuha**
this fixes #1004

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

File: articles/5-real-life-use-cases-of-5g-ultra-reliable-low-latency-communication-urllc/index.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+[5G](https://en.wikipedia.org/wiki/5G) adoption continues to progress at a high rate in subscribers and deployment. The 5th generation mobile network enables applications and services requiring better reliability, improved energy efficiency, massive connection density, and lower latency, making industries faster and efficient and our connected lives better.
+
+5G URLLC service makes it possible to support use cases that require very high reliability and extremely low latency. Examples of these use cases include industrial automation, intelligent transportation, smart electricity grid, entertainment support, and remote diagnosis and surgery.
```
Comment (lalith1403) [inline articles/5-real-life-use-cases-of-5g-ultra-reliable-low-latency-communication-urllc/index.md:3]: Expand the abbreviation, even though you have done it in the title!

File: articles/5-real-life-use-cases-of-5g-ultra-reliable-low-latency-communication-urllc/index.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+Smart grids improve electricity distribution worldwide by using communication capabilities to provide better balancing of power while detecting and mitigating faults.
+
+5G URLLC provides a seamless and affordable communication platform for deploying advanced technologies for managing power distribution networks in a time of rising need for flexibility and a growing amount of distributed energy resources.
+
+The Wireless for Verticals ([WIVE](https://wive.turkuamk.fi/)) research project presented one of the real-life use cases of 5G, relating to the electricity grid and harbor automation. Low latency and high reliability are significant in this project to ensure the protection of electricity grid infrastructure and harbor automation.
+
+Protection applications, including those in medium-voltage (MV) distribution networks, use the URLLC technology to release reliable, low-latency safety-related messages. This is ensuring immediate clearance of severe faults to keep the electricity distribution network running. In the long run, users benefit from increased safety of personnel and reduced damage to equipment.
+
+### Conclusion
```
Comment (lalith1403) [inline articles/5-real-life-use-cases-of-5g-ultra-reliable-low-latency-communication-urllc/index.md:65]: You can talk about the challenges as well in implementing these applications. The reason is, 5G is promising a future that you have clearly captured. However, the reader should also know about the challenges of implementing and succeeding in these applications.

**Review by lalith1403 — CHANGES_REQUESTED** [review]

**Comment by EKahuha** [comment]
@lalith1403 
 I have updated this article.

**Review by lalith1403 — APPROVED** [review]

**Comment by hectorkambow** [comment]
plagiarism check done

**Pull request merged**